sdk/keyvault/azure-security-keyvault-administration/CHANGELOG.md

```diff
@@ -1,14 +1,15 @@
 # Release History
 
-## 4.4.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 4.4.0 (2023-09-25)
 
 ### Other Changes
+- Explicitly added a `values()` method to all `ExpandableStringEnum` models:
```

Comment:
NIT: should we add azure-core and netty upgrade as usual what we had in auto-patch release